clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessor.java

```diff
@@ -121,6 +127,29 @@ public AuditLog.Builder auditLogBuilderWithJustification(
           Value.newBuilder().setStructValue(justificationStruct).build());
       auditLogBuilderCopy.setMetadata(metadataBuilder.build());
 
+      Claim justificationsClaim = jwt.getClaim("justs");
+
+      // Continue without populating RequestMetadata.RequestAttributes.Reason if the justifications
+      // claim is null or empty.
+      if (justificationsClaim.isNull()) {
+        log.warn("can't find 'justs' in claims");
+        return auditLogBuilderCopy;
+      }
+      List<Map> justificationsList = justificationsClaim.asList(Map.class);
+      if (justificationsList == null || justificationsList.isEmpty()) {
+        log.warn("justs in claims is null or empty");
+        return auditLogBuilderCopy;
+      }
+
+      RequestMetadata.Builder requestMetadataBuilder =
+          auditLogBuilderCopy.getRequestMetadataBuilder();
+      Request.Builder requestAttributesBuilder =
+          requestMetadataBuilder
+              .getRequestAttributesBuilder()
+              .setReason(new Gson().toJson(justificationsList));
+      requestMetadataBuilder =
+          requestMetadataBuilder.setRequestAttributes(requestAttributesBuilder);
+      auditLogBuilderCopy.setRequestMetadata(requestMetadataBuilder);
```

Comment:
What if there is an existing requestMetadata in the audit log?

Reply:
Context: https://github.com/abcxyz/lumberjack/pull/304#discussion_r972475115

Reply:
I think its worth adding a unit test for this.

Reply:
I will go back to the common way of get the object instead of builder since multiple people are having doubts on it.